src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -0,0 +1,729 @@
+-- This tests file includes tests for local execution of utility commands.
+-- For now, this file includes tests only for local execution of
+-- `TRUNCATE/DROP/DDL` commands for all kinds of distributed tables from
+-- the coordinator node having regular distributed tables' shards
+-- (shouldHaveShards = on) and having reference table placements in it.
+\set VERBOSITY terse
+SET citus.next_shard_id TO 1500000;
+SET citus.shard_replication_factor TO 1;
+SET citus.enable_local_execution TO ON;
+SET citus.shard_COUNT TO 32;
+SET citus.log_local_commands TO ON;
+CREATE SCHEMA local_commands_test_schema;
+SET search_path TO local_commands_test_schema;
+-- let coordinator have distributed table shards/placements
+SELECT 1 FROM master_add_node('localhost', :master_port, groupId => 0);
```

Comment:
make check-base EXTRA_TESTS="local_shard_utility_command_execution" is failing with the following:

```
--- /Users/onderkalaci/Documents/citus_code/citus/src/test/regress/expected/local_shard_utility_command_execution.out.modified  2020-03-13 08:09:3
9.000000000 +0100
+++ /Users/onderkalaci/Documents/citus_code/citus/src/test/regress/results/local_shard_utility_command_execution.out.modified   2020-03-13 08:09:3
9.000000000 +0100
@@ -6,20 +6,28 @@
 \set VERBOSITY terse
 SET citus.next_shard_id TO 1500000;
 SET citus.shard_replication_factor TO 1;
 SET citus.enable_local_execution TO ON;
 SET citus.shard_COUNT TO 32;
 SET citus.log_local_commands TO ON;
 CREATE SCHEMA local_commands_test_schema;
 SET search_path TO local_commands_test_schema;
 -- let coordinator have distributed table shards/placements
 SELECT 1 FROM master_add_node('localhost', :master_port, groupId => 0);
+NOTICE:  Replicating reference table "orders_reference" to the node localhost:57636
+NOTICE:  Replicating reference table "customer" to the node localhost:57636
+NOTICE:  Replicating reference table "nation" to the node localhost:57636
+NOTICE:  Replicating reference table "part" to the node localhost:57636
+NOTICE:  Replicating reference table "supplier" to the node localhost:57636
+NOTICE:  Replicating reference table "users_ref_test_table" to the node localhost:57636
+NOTICE:  Replicating reference table "events_reference_table" to the node localhost:57636
+NOTICE:  Replicating reference table "users_reference_table" to the node localhost:57636
  ?column? 
 ----------
         1
```

We should maybe set the log level just before `Local execution of TRUNCATE ` ?